contents/handbook/people/team-structure/ingestion.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Team Ingestion
+sidebar: Handbook
+showTitle: true
+hideAnchor: true
+---
+
+Team ingestion is part of the [Platform team](/handbook/people/team-structure/platform) at PostHog.
+
+## People
+
+- Tiina Turban (Full Stack Engineer)
+- Yakko Majuri (Full Stack Engineer)
+
+## Mission statement
+
+Provide the best events pipeline in the world.
+
+## Responsibilities
+
+Team Ingestion owns our ingestion pipeline end-to-end. That means we own client libraries, the Django server ingestion API, the ingestion (plugin) server, as well as our Kafka and ClickHouse setup, where it pertains to event ingestion.
+
+Our work generally falls into one of three categories:
+
+### Scaffolding to support core PostHog features
+
+In order to achieve company goals or introduce new features (often owned by other teams), changes to our ingestion pipeline may be required.
+
+An example of this is the work to remodel our events to store person and group data, which is essential to ensuring we can provide fast querying for users.
+
+While querying data is not owned by this team, the change to enable faster queries inevitably requires a large restructuring of our events pipeline, and thus we are owners of that component of the project.
+
+In short, a core responsibility of our team is to enable other teams to be successful.
+
+### Ingestion robustness
+
+On the road to providing the best events pipeline in the world, we need to build a system that is robust. 
+
+To do so, we must ensure:
+
+- **Reliability:** We should not lose events and events ingested should be correct
+- **Scalability:** We should be able to scale to massive event volumes
+- **Maintainability:** It should be easy to debug and contribute to our ingestion pipeline
```

Comment:
Somewhere here we should also mention the goal that any oncall at PostHog or a self-hosted user should be be able to do initial investigation into ingestion problems (as in the goal we have that upon ingestion problems the first thought isn't lets robodial Yakko) Tied to the goal we have, that need work and then keeping up to date:
Ingestion monitoring and management
    - Runbooks, dashboards, and alerts in Grafana on cloud and on self hosted

Reply:
Yeah so that falls under more specific goals rather than responsibilities.

It is part of "Ingestion monitoring and management" and in some ways implied via "maintainability" too.

I'd rather keep this section more general